tsconfig.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Including this wouldn't cause conflict with #1 ?
Or there is some migration order

Reply:
I copied this tsconfig from #1 so if we can get all of those changes merged in I don’t expect to see any conflicts (but if there are I can resolve them).

Reply:
👌